adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMRDDFunctions.scala

```diff
@@ -64,6 +64,12 @@ trait ADAMSaveAnyArgs extends SaveArgs {
    * @see asSingleFile
    */
   var deferMerging: Boolean
+
+  /**
+   * If asSingleFile is true and deferMerging is false, disables the use of the
+   * fast file concatenation engine.
+   */
+  var disableFastConcat: Boolean
```

Comment:
Per this, #1295, and #1438, perhaps we should brainstorm on how to clean up the save methods and ensure consistency.  One or more enums, better SaveArgs classes, a context map, etc.